config/configuration.js

```diff
@@ -35,8 +35,8 @@ module.exports = {
   appSecret: process.env.ANYFETCH_API_SECRET,
 
   box: {
-    api: process.env.BOX_API_ID,
-    secret: process.env.BOX_API_SECRET
+    api: process.env.BOX_API_ID || 'api_id',
```

Comment:
So the tests can be run without having to define any env variables, should it be removed?